src/Sulu/Component/Content/Types/Rlp/Mapper/PhpcrMapper.php

```diff
@@ -593,9 +593,7 @@ private function checkResourceLocator(NodeInterface $routeNode, $resourceLocator
                 // route already exists and referenced on contentNode
                 return true;
             } else {
-                throw new ResourceLocatorAlreadyExistsException(sprintf(
-                    'Resource locator "%s" already exists. Route node at path "%s"', $resourceLocator, $routeNode->getPath()
-                ));
+                throw new ResourceLocatorAlreadyExistsException($resourceLocator, $routeNode->getPath());
```

Comment:
But then the developer catching the exception can't be sure to get a value when calling the getters on this exception, or am I missing something?